console/atest-ui/src/views/TestCase.vue

```diff
@@ -124,9 +134,19 @@ const handleTestResult = (e: any) => {
     isResponseFile.value = true
   } else if(e.body !== ''){
     testResult.value.bodyLength = e.body.length
-    testResult.value.bodyObject = JSON.parse(e.body);
-    testResult.value.originBodyObject = JSON.parse(e.body);
-    responseBodyFilter()
+    try {
+      // Try to parse as JSON, fallback to plain text if parsing fails
+      testResult.value.bodyObject = JSON.parse(e.body);
+      testResult.value.originBodyObject = JSON.parse(e.body);
+      responseBodyFilter()
+    } catch (error) {
+      // This is an expected case for non-JSON responses (like text/plain)
+      // We intentionally display as plain text instead of attempting JSON parsing
+      console.debug("Response body is not valid JSON, displaying as plain text:", error);
+      testResult.value.bodyText = e.body;
+      testResult.value.bodyObject = null;
```

Comment:
Type 'null' is not assignable to type '{}'.ts-plugin(2322)

Reply:
You can check the content type, then handle JSON or not.

![image](https://github.com/user-attachments/assets/cc476cc2-fef3-4e8a-b145-464c2c858832)